src/parse.coffee

```diff
@@ -33,8 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
-      apiElements.content = apiElements.content.concat(annotations)
+      apiElements.content.unshift(warning) if warning
```

Comment:
So I tried it and it doesn't seem to work. The line

```coffee
apiElements.unshift(warning) if warning
```

Breaks the build and doesn't seem to add the warning at all, according to the diffs in failing tests.